sample_apps_reinvent2021/go/scheduled-query-sample.go

```diff
@@ -0,0 +1,261 @@
+			fmt.Printf("ErrorReportLocation Details: %+v\n", response.Message.ScheduledQueryRunSummary.ErrorReportLocation)
+			s3BucketName := response.Message.ScheduledQueryRunSummary.ErrorReportLocation.S3ReportLocation.BucketName
+			objectKey := response.Message.ScheduledQueryRunSummary.ErrorReportLocation.S3ReportLocation.ObjectKey
+			if s3BucketName != "" && objectKey != "" {
+				timestreamDependencyHelper.ParseS3ErrorReport(s3BucketName, objectKey)
+			}
+		case utils.SCHEDULED_QUERY_DELETED:
+			fallthrough
+		case utils.SCHEDULED_QUERY_UPDATED:
+			fallthrough
+		default:
+			fmt.Printf("response = %+v\n", response)
+		}
+
+		if (didQuerySucceedManually && wasQueryTriggeredAsExpected) || queryFailed {
+			break
+		}
+
+	}
+
+	OUTPUT_FILE := "output.log"
+	var f *os.File
+	if OUTPUT_FILE != "" {
+		var ferr error
+		f, ferr = os.Create(OUTPUT_FILE)
+		utils.HandleError(ferr, "", true)
+		defer f.Close()
+	}
+
+	if wasQueryTriggeredAsExpected || didQuerySucceedManually {
+		fmt.Println("Fetching Scheduled Query execution results")
+		queryString := fmt.Sprintf("SELECT * FROM %s.%s", sqResultsDatabaseName, sqResultsTableName)
+		utils.RunQuery(&queryString, timestreamBuilder.QuerySvc, f)
+	}
+
+	err = timestreamBuilder.UpdateScheduledQuery(scheduledQueryArn)
+	utils.HandleError(err, fmt.Sprintf("Failed to update scheduled query with scheduledQueryArn %s ", scheduledQueryArn), false)
+
+	if *deleteResourcesFlag {
+
+		reader := bufio.NewReader(os.Stdin)
+		fmt.Println("deleteResourcesFlag is set to true, Deleting all the resources that got created " +
+			" as part of this sample, hit any key to continue")
+		reader.ReadString('\n')
+
+		if scheduledQueryArn != "" {
+			timestreamBuilder.DeleteScheduledQuery(scheduledQueryArn)
+		}
+
+		scheduledQueries, err = timestreamBuilder.ListScheduledQueries()
+		if err == nil {
+			fmt.Printf("scheduledQueries Size after deleting current ScheduledQuery: %d\n", len(scheduledQueries))
+		}
+
+		if s3BucketName != "" {
+			timestreamDependencyHelper.DeleteS3Bucket(s3BucketName)
+		}
+
+		if policyArn != "" {
+			timestreamDependencyHelper.DetachIamPolicy(utils.ROLE_NAME, policyArn)
+			timestreamDependencyHelper.DeleteIamPolicy(policyArn)
+			timestreamDependencyHelper.DeleteIamRole(utils.ROLE_NAME)
+		}
+
+		if subscriptionArn != "" {
+			timestreamDependencyHelper.UnsubscribeToSnsTopic(subscriptionArn)
+			queueUrl, err := timestreamDependencyHelper.GetSqsQueryUrl(sqSampleAppQueueName)
+			if err == nil && queueUrl != "" {
+				timestreamDependencyHelper.DeleteSqsQueue(queueUrl)
+			}
+		}
+
+		if topicArn != "" {
+			timestreamDependencyHelper.DeleteSnsTopic(topicArn)
+		}
+
+		timestreamBuilder.DeleteTable(sqResultsDatabaseName, sqResultsTableName)
+		timestreamBuilder.DeleteDatabase(sqResultsDatabaseName)
+		timestreamBuilder.DeleteTable(databaseName, tableName)
+		timestreamBuilder.DeleteDatabase(databaseName)
+	}
```

Comment:
can we print a message like 'Please delete the following resources' and list the arns of the all the resources in an else block

Reply:
hmm good idea.   let me do this for Go and later we can make it for other languages too.